apps/swirl-docs/src/pages/sitemap.xml.ts

```diff
@@ -0,0 +1,52 @@
+import { navItems } from "@swirl/lib/navigation";
+
+function generateSiteMap(paths: string[]) {
+  const baseUrl = "https://getflip.dev";
+
+  return `<?xml version="1.0" encoding="UTF-8"?>
+   <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+     ${paths
+       .map((path) => {
+         return `
+       <url>
+           <loc>${`${baseUrl}${path}`}</loc>
+           <changefreq>weekly</changefreq>
+           <priority>0.7</priority>
```

Comment:
**question:** Can this be removed if all pages get the same priority anyway?

Reply:
we could. i looked on different comparable sites and a few had 0.7 prio on their sites as well. then i googled a bit and found this [article](https://www.contentpowered.com/blog/xml-sitemap-priority-changefreq/#sitemap_priority_attribute) that explained the values in a little bit more depth, but as this param is optional i am totally fine with removing it

Reply:
That's fine. Thanks.